src/redux/investment/reducer.js

```diff
@@ -0,0 +1,58 @@
+import {
+  FETCH_USER_INVESTMENT_SHARES_START,
+  FETCH_USER_INVESTMENT_SHARES_FAIL,
+  FETCH_USER_INVESTMENT_SHARES_SUCCESS,
+  FETCH_STOCK_PRICE_START,
+  FETCH_STOCK_PRICE_FAIL,
+  FETCH_STOCK_PRICE_SUCCESS,
+} from './actionType';
+
+const initState = {
+  isLoading: false,
+  userShares: undefined,
+  price: undefined,
+  error: '',
```

Comment:
error should be an object (we usually invoke error.message)

Reply:
Please discuss this with the team, we have to be consequent